EstateAdministrationUI/Areas/Estate/Controllers/OperatorController.cs

```diff
@@ -0,0 +1,148 @@
+
+                // All good with model, call the client to create the operator
+                CreateOperatorResponseModel createOperatorResponse =
+                    await this.ApiClient.CreateOperator(accessToken, this.User.Identity as ClaimsIdentity, createOperatorModel, cancellationToken);
+
+                // Operator Created, redirect to the Operator List screen
+                return this.RedirectToAction("GetOperatorList",
+                                             "Operator",
+                                             new
+                                             {
+                                                 Area = "Estate"
+                                             });
+            }
+
+            // If we got this far, something failed, redisplay form
+            return this.View("CreateOperator", viewModel);
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetOperatorList(CancellationToken cancellationToken)
+        {
+            return this.View("OperatorList");
+        }
+
+        /// <summary>
+        /// Gets the merchant list as json.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns></returns>
+        [HttpPost]
+        public async Task<IActionResult> GetOperatorListAsJson(CancellationToken cancellationToken)
+        {
+            Logger.LogDebug("In method GetOperatorListAsJson");
+
+            // Search Value from (Search box)  
+            String searchValue = this.HttpContext.Request.Form["search[value]"].FirstOrDefault();
+            Logger.LogDebug($"searchvalue is {searchValue}");
+
+            String accessToken = await this.HttpContext.GetTokenAsync("access_token");
+            Logger.LogDebug("got access token");
+
+            EstateModel estate = await this.ApiClient.GetEstate(accessToken, this.User.Identity as ClaimsIdentity, cancellationToken);
+
+            List<OperatorListViewModel> operatorViewModels = this.ViewModelFactory.ConvertFrom(estate.EstateId, estate.Operators);
+
+            Logger.LogDebug($"operator list count is {operatorViewModels.Count}");
+
+            Expression<Func<OperatorListViewModel, Boolean>> whereClause = m => m.OperatorName.Contains(searchValue, StringComparison.OrdinalIgnoreCase);
+
+            DataTablesResult<OperatorListViewModel> dataTableResult = Helpers.GetDataForDataTable(this.Request.Form, operatorViewModels, whereClause);
+
+            String jsonResult = JsonConvert.SerializeObject(dataTableResult);
+            Logger.LogDebug(jsonResult);
+
+            return this.Json(Helpers.GetDataForDataTable(this.Request.Form,operatorViewModels, whereClause));
+        }
+
+        /// <summary>
+        /// Validates the model.
+        /// </summary>
+        /// <param name="viewModel">The view model.</param>
+        /// <returns></returns>
+        private Boolean ValidateModel(CreateOperatorViewModel viewModel)
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'viewModel'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=6070534)